internal/httputil/response_recorder.go

```diff
@@ -0,0 +1,25 @@
+package httputil
+
+import "net/http"
+
+type ResponseRecorder struct {
```

Comment:
Decorator may be redundant. Check Header() method of http.ResponseWriter interface.

Reply:
I didn't find anything in the Header()

Reply:
Ok, thanks for checking.